tests/test_media_handlers.py

```diff
@@ -1,13 +1,80 @@
+from functools import partial
+import io
+import json
+
+import mujson
 import pytest
+import ujson
 
 from falcon import media
+from falcon.util import compat
+
+if compat.PY3:
+    import rapidjson
+    import orjson
+
+
+COMMON_SERIALIZATION_PARAM_LIST = [
+    # Default json.dumps, with only ascii
+    (None, {'test': 'value'}, b'{"test":"value"}'),
+    (mujson.dumps, {'test': 'value'}, b'{"test":"value"}'),
+    (ujson.dumps, {'test': 'value'}, b'{"test":"value"}'),
+    (partial(lambda media, **kwargs: json.dumps([media, kwargs]),
+     ensure_ascii=True),
+     {'test': 'value'},
+     b'[{"test":"value"},{"ensure_ascii":true}]'),
+]
+
+COMMON_DESERIALIZATION_PARAM_LIST = [
+    (None, b'[1, 2]', [1, 2]),
+    (partial(json.loads,
+             object_hook=lambda data: {k: v.upper() for k, v in data.items()}),
+     b'{"key": "value"}',
+     {'key': 'VALUE'}),
+
+    (mujson.loads, b'{"test": "value"}', {'test': 'value'}),
+    (ujson.loads, b'{"test": "value"}', {'test': 'value'}),
+]
+
+YEN = b'\xc2\xa5'
+
+if compat.PY3:
```

Comment:
If I were to write this from scratch, I would probably define ``skip`` based on the Python version, but it is mostly a matter of taste, not a requirement from me.

Reply:
[fixed]